### PR TITLE
Added L2circuit rule and playbook

### PR DESCRIPTION
**Pull request by vvikramb**
Added below files :

check-l2circuit-pw-state.rule
L2CircuitConnectionInfo 2.yml
l2circuit-kpis.playbook

### CODE REVIEW

**Review by gvarao — APPROVED** [review]

File: juniper_official/vpn/l2vpn/L2CircuitConnectionInfo 2.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+L2CircuitConnectionNbrView:
+    fields:
+      view: L2CircuitConnectionIdTable
+L2CircuitConnectionIdTable:
+      item: connection
+      key: connection-id
+      view: L2CircuitConnectionIdView
+L2CircuitConnectionIdView:
+  fields:
+        connection-status: connection-status
```
Comment (gaddevijay) [inline juniper_official/vpn/l2vpn/L2CircuitConnectionInfo 2.yml:16]: xslt should be added

File: juniper_official/vpn/l2vpn/check-l2circuit-pw-state.rule [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            field neighbor {
+                sensor l2ckt {
+                    where "neighbor-address =~ /{{neighbor-address}}/";
+                    path neighbor-address;
+                }
+                type string;
+                description "l2circuit neighbor address";
+            }
+            field connection-status {
+                sensor l2ckt {
```
Comment (gaddevijay) [inline juniper_official/vpn/l2vpn/check-l2circuit-pw-state.rule:35]: enum should be included

File: juniper_official/vpn/l2vpn/check-l2circuit-pw-state.rule [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            }
+            rule-properties {
+                version 1;
+                contributor juniper;
+                supported-healthbot-version 4.0.0;
+                supported-devices {
+                    juniper {
+                         operating-system junos {
+                            products MX {
+                                platforms All {
```
Comment (gaddevijay) [inline juniper_official/vpn/l2vpn/check-l2circuit-pw-state.rule:83]: All is not valid

File: juniper_official/vpn/l2vpn/check-l2circuit-pw-state.rule [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                                    releases 15.1R1 {
+                                        release-support min-supported-release;
+                                    }
+                                }
+                            }
+                         }
+                         operating-system junosEvolved {
+                            products ACX {
+                                platforms All {
```
Comment (gaddevijay) [inline juniper_official/vpn/l2vpn/check-l2circuit-pw-state.rule:92]: All is not valid

File: juniper_official/vpn/l2vpn/l2circuit-kpis.playbook [hunk 5]
```diff
@@ -0,0 +1,15 @@
+/*
+ * Playbook contains l2circuit realted KPIs
+ *
+ * Rule vpn.l2circuit/check-l2circuit-pw-state - Checks status of l2circuit
+ * on given pseudowire and notifies anamolies based on Discovery State
+ *
+ */
+
+healthbot {
+    playbook l2circuit-kpis {
```
Comment (gaddevijay) [inline juniper_official/vpn/l2vpn/l2circuit-kpis.playbook:10]: Playbooks should not be checked in

**Pull request merged**